### ISSUE TITLE
WASM isn't being served from Cloudfront with `application/wasm` content type

### ISSUE THREAD

**Issue by jameslittle230** [jameslittle230/stork#249]
Full error:

```
stork.wasm is returned with the wrong content type on stork-search so we fallback to WebAssembly.instantiate
```

**Comment by ArsenArsen** [comment]
I believe the CDN would need to return a `application/wasm` content type instead of an octet-stream to fix this.

**Comment by jameslittle230** [comment]
Yup, definitely. I think the missing context here is that I'm specifying an `application/wasm` content type in my [script that uploads files to the CDN](https://github.com/jameslittle230/stork/blob/master/scripts/upload_build_artifacts.py#L66), but that content type isn't being plumbed through the infrastructure correctly.

If I had to guess, it's something goofy with my Cloudfront setup, but I have to do some more investigation to check.
